### PR TITLE
Add note about PullSecretName into builds

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Comment by mfojtik** [comment]
@bparees @adellape this is related to https://trello.com/c/HMtoPxYw/567-5-add-ability-to-specify-credentials-sti-should-use-when-pulling-images-build-codefreeze

File: architecture/core_objects/builds.adoc [hunk 1]
```diff
@@ -166,6 +167,24 @@ Once you have the `*Secret*` created, you can add a `PushSecretName` field into
 `Output` section inside the `BuildConfig` and set it to the name of the
 `*Secret*` that you created, in this case `*dockerhub*`.
 
+For pulling the builder Docker image from private Docker registry, you have to specify
```
Comment (bparees) [inline architecture/core_objects/builds.adoc]: Can you go ahead and add the sample for PushSecretName too?  you can make it one sample if you want.

File: architecture/core_objects/builds.adoc [hunk 2]
```diff
@@ -97,12 +97,13 @@ leverage for your applications.
...
 
-[#using-docker-credentials-for-pushing-images]
-== Using Docker Credentials for Pushing Images
+[#using-docker-credentials-for-pushing-and-pulling-images]
+== Using Docker Credentials for Pushing and Pulling Images
 
-In case you want to push the output image into private Docker registry that
-requires authentication or Docker Hub, you must supply a `.dockercfg` file
-with valid Docker registry credentials.
+In case you want to push the output image into private Docker Registry or pull
```
Comment (adellape) [inline architecture/core_objects/builds.adoc]: s/into private/into a private/ ?

File: architecture/core_objects/builds.adoc (hunk 1 above)
Comment (adellape) [inline architecture/core_objects/builds.adoc]: s/from private/from a private/ ?

**Comment by mfojtik** [comment]
@adellape @bparees for some reason my Github prevents me to push into this branch, so closing this PR and moving to a new one:

https://github.com/openshift/openshift-docs/pull/398

**Pull request closed without merging**